test/unit/blur.js

```diff
@@ -91,4 +104,33 @@ describe('Blur', function () {
           });
       });
   });
+
+  it('invalid precision', function () {
+    assert.throws(function () {
+      sharp(fixtures.inputJpg).blur({ sigma: 1, precision: 'invalid' });
+    });
```

Comment:
Could add an assertion for the error message here.
```suggestion
    }, /Expected one of: integer, float, approximate for precision but received invalid of type string/);
```